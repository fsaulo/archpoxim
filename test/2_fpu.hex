0xDC000015
0xFC000000
0xFC000000
0xFC000000
0xFC000000
0xDC000002
0xDC000001
0xDC000000
0x2BE32042
0x0B5A0000
0x0B7B0000
0x68210000
0x68420000
0x68630000
0x60840000
0x00C00001
0x2C2317C4
0x80000000
0x5C060000
0xBBFFFFFE
0x00C00000
0x7C000000
0x03C07FFC
0x87E10001
0x68200045
0x68400046
0x68600047
0x68800048
0x00A00001
0x74A10000
0x74A20000
0x00A00001
0x6CA40000
0xE7FFFFF0
0x00A00005
0x6CA40000
0xE7FFFFED
0x00A00002
0x6CA40000
0xE7FFFFEA
0x00A00009
0x74A20000
0x00A00003
0x6CA40000
0xE7FFFFE5
0x00A00006
0x6CA40000
0xE7FFFFE2
0x00A00004
0x6CA40000
0xE7FFFFDF
0x00A00000
0x74A20000
0x00A00004
0x6CA40000
0xE7FFFFDA
0x00A00007
0x6CA40000
0xE7FFFFD7
0x00A00008
0x6CA40000
0xE7FFFFD4
0x00A00009
0x6CA40000
0xE7FFFFD1
0x00A00ABC
0x6CA40000
0xE7FFFFCE
0xFC000000
0x20202220
0x20202221
0x20202222
0x8080888F