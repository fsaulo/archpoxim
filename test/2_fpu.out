[START OF SIMULATION]
0x00000000:	bun 21                   	PC=0x00000058
0x00000058:	mov sp,32764             	SP=0x00007FFC
0x0000005C:	sbr sr[1]                	SR=0x00000002
0x00000060:	l32 r1,[r0+69]           	R1=MEM[0x00000114]=0x20202220
0x00000064:	l32 r2,[r0+70]           	R2=MEM[0x00000118]=0x20202221
0x00000068:	l32 r3,[r0+71]           	R3=MEM[0x0000011C]=0x20202222
0x0000006C:	l32 r4,[r0+72]           	R4=MEM[0x00000120]=0x8080888F
0x00000070:	mov r5,1                 	R5=0x00000001
0x00000074:	s32 [r1+0],r5            	MEM[0x80808880]=R5=0x00000001
0x00000078:	s32 [r2+0],r5            	MEM[0x80808884]=R5=0x00000001
0x0000007C:	mov r5,1                 	R5=0x00000001
0x00000080:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x01
0x00000084:	call -16                 	PC=0x00000048,MEM[0x00007FFC]=0x00000088
[HARDWARE INTERRUPTION 3]
0x00000018:	bun 1                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x00000084,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x00000001
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000001
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x40000000
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x00000088
0x00000088:	mov r5,5                 	R5=0x00000005
0x0000008C:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x05
0x00000090:	call -19                 	PC=0x00000048,MEM[0x00007FFC]=0x00000094
[HARDWARE INTERRUPTION 4]
0x0000001C:	bun 0                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x00000090,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000001
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x40000000
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x00000094
0x00000094:	mov r5,2                 	R5=0x00000002
0x00000098:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x02
0x0000009C:	call -22                 	PC=0x00000048,MEM[0x00007FFC]=0x000000A0
0x00000048:	cmpi r6,0                	SR=0x00000042
[HARDWARE INTERRUPTION 3]
0x00000018:	bun 1                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000042}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x00000048,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000001
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x3F800000
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000042,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x0000004C
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000A0
0x000000A0:	mov r5,9                 	R5=0x00000009
0x000000A4:	s32 [r2+0],r5            	MEM[0x80808884]=R5=0x00000009
0x000000A8:	mov r5,3                 	R5=0x00000003
0x000000AC:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x03
0x000000B0:	call -27                 	PC=0x00000048,MEM[0x00007FFC]=0x000000B4
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
[HARDWARE INTERRUPTION 3]
0x00000018:	bun 1                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000042}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x0000004C,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000009
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x41900000
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000042,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000B4
0x000000B4:	mov r5,6                 	R5=0x00000006
0x000000B8:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x06
0x000000BC:	call -30                 	PC=0x00000048,MEM[0x00007FFC]=0x000000C0
[HARDWARE INTERRUPTION 4]
0x0000001C:	bun 0                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x000000BC,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x41900000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x41900000
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000C0
0x000000C0:	mov r5,4                 	R5=0x00000004
0x000000C4:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x04
0x000000C8:	call -33                 	PC=0x00000048,MEM[0x00007FFC]=0x000000CC
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
[HARDWARE INTERRUPTION 3]
0x00000018:	bun 1                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000042}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x00000048,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x41900000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x3DE38E39
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000042,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x0000004C
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000CC
0x000000CC:	mov r5,0                 	R5=0x00000000
0x000000D0:	s32 [r2+0],r5            	MEM[0x80808884]=R5=0x00000000
0x000000D4:	mov r5,4                 	R5=0x00000004
0x000000D8:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x04
0x000000DC:	call -38                 	PC=0x00000048,MEM[0x00007FFC]=0x000000E0
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000042
0x0000004C:	beq -2                   	PC=0x00000048
[HARDWARE INTERRUPTION 2]
0x00000014:	bun 2                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000042}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x0000004C,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x3DE38E39
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x20
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000042,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000E0
0x000000E0:	mov r5,7                 	R5=0x00000007
0x000000E4:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x07
0x000000E8:	call -41                 	PC=0x00000048,MEM[0x00007FFC]=0x000000EC
[HARDWARE INTERRUPTION 4]
0x0000001C:	bun 0                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x000000E8,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x00000001
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000EC
0x000000EC:	mov r5,8                 	R5=0x00000008
0x000000F0:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x08
0x000000F4:	call -44                 	PC=0x00000048,MEM[0x00007FFC]=0x000000F8
[HARDWARE INTERRUPTION 4]
0x0000001C:	bun 0                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x000000F4,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x00000001
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x000000F8
0x000000F8:	mov r5,9                 	R5=0x00000009
0x000000FC:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0x09
0x00000100:	call -47                 	PC=0x00000048,MEM[0x00007FFC]=0x00000104
[HARDWARE INTERRUPTION 4]
0x0000001C:	bun 0                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x00000100,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x00000001
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x00
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x00000104
0x00000104:	mov r5,2748              	R5=0x00000ABC
0x00000108:	s8 [r4+0],r5             	MEM[0x8080888F]=R5=0xBC
0x0000010C:	call -50                 	PC=0x00000048,MEM[0x00007FFC]=0x00000110
[HARDWARE INTERRUPTION 2]
0x00000014:	bun 2                    	PC=0x00000020
0x00000020:	push r1,r2,r3,r4,sr      	MEM[0x00007FEC]{0x20202220,0x20202221,0x20202222,0x8080888F,0x00000002}={R1,R2,R3,R4,SR}
0x00000024:	add cr,cr,r0             	CR=CR+R0=0x01EEE754,SR=0x00000002
0x00000028:	add ipc,ipc,r0           	IPC=IPC+R0=0x0000010C,SR=0x00000002
0x0000002C:	l32 r1,[r1+0]            	R1=MEM[0x80808880]=0x40000000
0x00000030:	l32 r2,[r2+0]            	R2=MEM[0x80808884]=0x00000000
0x00000034:	l32 r3,[r3+0]            	R3=MEM[0x80808888]=0x00000001
0x00000038:	l8 r4,[r4+0]             	R4=MEM[0x8080888F]=0x20
0x0000003C:	mov r6,1                 	R6=0x00000001
0x00000040:	pop sr,r4,r3,r2,r1       	{SR,R4,R3,R2,R1}=MEM[0x00007FD8]{0x00000002,0x8080888F,0x20202222,0x20202221,0x20202220}
0x00000044:	reti                     	IPC=MEM[0x00007FF0]=0x00000000,CR=MEM[0x00007FF4]=0x00000000,PC=MEM[0x00007FF8]=0x00000048
0x00000048:	cmpi r6,0                	SR=0x00000002
0x0000004C:	beq -2                   	PC=0x00000050
0x00000050:	mov r6,0                 	R6=0x00000000
0x00000054:	ret                      	PC=MEM[0x00007FFC]=0x00000110
0x00000110:	int 0                    	CR=0x00000000,PC=0x00000000
[END OF SIMULATION]
